test/acceptance/support/files/SITISFI0001_AP_20220908125135717.dat

```diff
@@ -0,0 +1,3 @@
+B^2022-09-08^1^1000^0001^SFI^AP
+H^SFI0000001^01^00000001^1^1000000001^GBP^1000^RP00^GBP^SFI^Q4
+L^SFI0000001^1000^2022^80101^DRD10^00000001^RP00^N^1^G00 - Gross value of claim^2022-12-01^2022-12-01^SOS273
```

Comment:
Suggest moving this file to the existing `test/files` directory, rather than creating a second location with test files.